src/main/java/net/dv8tion/jda/core/managers/ChannelManager.java

```diff
@@ -367,6 +367,82 @@ public ChannelManager removePermissionOverride(IPermissionHolder permHolder)
         return this;
     }
 
+    /**
+     * Syncs all {@link net.dv8tion.jda.core.entities.PermissionOverride PermissionOverrides} of this Channel with
+     * its parent ({@link net.dv8tion.jda.core.entities.Category Category}).
+     * <br/>After this operation, all {@link net.dv8tion.jda.core.entities.PermissionOverride PermissionOverrides}
+     * will be exactly the same as the ones from the syncSource.
+     * <br/>This behaves as if calling {@link #sync(Channel)} with this Channel's {@link net.dv8tion.jda.core.entities.Channel#getParent() Parent}.
+     *
+     * @throws  java.lang.IllegalStateException
+     *          If this Channel has no parent
+     * @throws  net.dv8tion.jda.core.exceptions.InsufficientPermissionException
+     *          If the currently logged in account does not have {@link net.dv8tion.jda.core.Permission#MANAGE_PERMISSIONS Permission.MANAGE_PERMISSIONS}
+     *          in this channel
+     *
+     * @return  ChannelManager for chaining convenience
+     */
+    @CheckReturnValue
+    public ChannelManager sync()
+    {
+        if(channel.getParent() == null)
+            throw new IllegalStateException("sync() requires a parent category");
+        return sync(channel.getParent());
+    }
+
+    /**
+     * Syncs all {@link net.dv8tion.jda.core.entities.PermissionOverride PermissionOverrides} of this Channel with
+     * the given ({@link net.dv8tion.jda.core.entities.Channel Channel}).
+     * <br/>After this operation, all {@link net.dv8tion.jda.core.entities.PermissionOverride PermissionOverrides}
+     * will be exactly the same as the ones from the syncSource.
+     * <br/>This will only work for Channels of the same {@link net.dv8tion.jda.core.entities.Guild Guild}.
+     *
+     * @param   syncSource
+     *          The Channel from where all PermissionOverrides should be copied from
+     *
+     * @throws  java.lang.IllegalArgumentException
+     *          If the given snySource is {@code null}, this Channel or from a different Guild.
+     * @throws  net.dv8tion.jda.core.exceptions.InsufficientPermissionException
+     *          If the currently logged in account does not have {@link net.dv8tion.jda.core.Permission#MANAGE_PERMISSIONS Permission.MANAGE_PERMISSIONS}
+     *          in this channel
+     *
+     * @return  ChannelManager for chaining convenience
+     */
+    @CheckReturnValue
+    public ChannelManager sync(Channel syncSource)
+    {
+        Checks.notNull(syncSource, "SyncSource");
+        Checks.check(channel.getGuild().equals(syncSource.getGuild()), "Sync only works for channels of same guild");
+        Checks.check(!channel.equals(syncSource), "Cannot sync with itself");
```

Comment:
We could just return here and do nothing instead of failing?

Reply:
Why do it differently here compared to all other methods in jda checking arguments. i think we shouldn't silently do nothing in case of bad input

Reply:
Well, its not bad input its a trivial case.

Reply:
ah i guess you meant only the self part... which im ok with... makes sense